src/name_bazaar/ui/components/name_bazaar_meta_tags.cljs

```diff
@@ -0,0 +1,20 @@
+(ns name-bazaar.ui.components.name-bazaar-meta-tags
```

Comment:
Should be named `name-bazaar.ui.components.meta-tags`, no need to explicitly say name-bazaar